pkg/controller/config/config.go

```diff
@@ -15,22 +45,38 @@ var (
 	configSection = config.MustRegisterSection(configSectionKey, defaultConfig)
 
 	defaultConfig = &Config{
-		MaxWorkflowRetries:  5,
+		Workers: 20,
+		WorkflowReEval: config.Duration{
+			Duration: 10 * time.Second,
+		},
+		DownstreamEval: config.Duration{
+			Duration: 30 * time.Second,
+		},
+		MaxWorkflowRetries:  10,
 		MaxDatasetSizeBytes: 10 * 1024 * 1024,
 		Queue: CompositeQueueConfig{
-			Type: CompositeQueueSimple,
+			Type: CompositeQueueBatch,
+			BatchingInterval: config.Duration{
+				Duration: time.Second,
+			},
+			BatchSize: -1,
```

Comment:
What does -1 mean?

Reply:
all elements in the batch (infinite) this is what we have set everywhere